src/doc/trpl/error-handling.md

```diff
@@ -181,6 +181,8 @@ match version {
 This function makes use of an enum, `ParseError`, to enumerate the various
 errors that can occur.
 
+The [`Debug`](../core/fmt/trait.Debug.html) trait is what let's us print the enum value using the `{:?}` format operation.
```

Comment:
Ah! so, https://github.com/rust-lang/rust/pull/25377 failed. so no, you don't need to make a new PR